private_repo/ansible/inventory/dev/KnowledgePlatform/common.yml

```diff
@@ -159,11 +159,11 @@ cloud_artifact_storage_namespace: "{{ cloud_public_storage_namespace }}"
 # cloud_storage_redisbackup_bucketname: ""
 
 # Building block vars
-cloud_storage_base_url: "{{ cloud_storage_url }}"
-cloudstorage_base_path: "{{ cloud_storage_url }}"
-valid_cloudstorage_base_urls: '["{{ cloud_storage_url }}"]'
+cloud_storage_base_url: "https://{{ cloud_storage_url }}"
```

Comment:
@ddevadat For Azure, we dont give https as prefix. Is this only specific to OCI?

Reply:
corrected this